winui/Masked-TextBox/Formatting-Value.md

```diff
@@ -7,4 +7,135 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Value mask formats in WinUI SfMaskedTextBox
\ No newline at end of file
+# Value mask formats in WinUI SfMaskedTextBox
+
+`SfMaskedTextBox` allows you to format the characters in the `Value` property in a mask scenario (when the Mask property is set). By default, the `Value` property holds your input characters, prompt characters and the literals defined in the mask. You can modify this and allow the `Value` property to hold the characters without prompt and literals by setting the `ValueMaskFormat` property of the control. The `Value` in the `SfMaskedTextBox` is formatted by any one of the following formatting enum values:
```

Comment:
@Vignesh-SF3580 

The MaskedTextBox control allows you to format input values with prompt and literal characters which are defined in the mask expression by setting the ValueMaskFormat property. By default, the Value property of the control includes the characters typed by the user, as well as any prompt or literal characters in the input. The control provides several formatting options, which are listed below.

Reply:
Changed the content for formatting value like this.